ui/src/shared/components/HistogramChart.tsx

```diff
@@ -34,6 +35,7 @@ interface Props {
   colors: HistogramColor[]
   colorScale: ColorScale
   onZoom: (TimePeriod) => void
+  onSortChartBars: () => SortFn
```

Comment:
Can this be passed just as a `SortFn` rather than a `() => SortFn`? It is confusing to me to accept data via a callback. Usually I think of `onFoo` props as emitting a request to perform side effects.

Reply:
yeppers